client/hotrod-client/src/test/java/org/infinispan/client/hotrod/test/MultiHotRodServersTest.java

```diff
@@ -65,14 +65,19 @@ protected RemoteCacheManager createClient(int i) {
             .host(host)
             .port(serverPort)
             .maxRetries(maxRetries())
-            .contextInitializer(contextInitializer());
+            .contextInitializers(contextInitializers());
       return clientBuilder;
    }
 
    protected SerializationContextInitializer contextInitializer() {
       return null;
    }
 
+   protected SerializationContextInitializer[] contextInitializers() {
+      SerializationContextInitializer sci = contextInitializer();
+      return sci == null ? new SerializationContextInitializer[0] : new SerializationContextInitializer[]{sci};
```

Comment:
I'd rather remove `contextInitializer()` and make this method return a `List`, so subclasses can do `return Arrays.asList(...)`.